### DISCUSSION TITLE
Read XY file from XRD data

### DISCUSSION THREAD

**Discussion by bbellonl** [mtex-toolbox/mtex#1054] [Feature Requests]
I want to work with some xrd data but there is no way to read such a file, which is quite simple because it only has intensity and the angle.

**Comment by ralfHielscher** [comment]
Actually it is quite straight forward to import such data using the command [`loadPoleFigure_generic`](https://mtex-toolbox.github.io/loadPoleFigure_generic.html).

Ralf.

PS: the command leads you to the documentation.

**Reply by bbellonl** [reply]
In fact I cannot use that one because I only have the theta angle, I only have the polar angle and the intensity, not the azimuthal.
Thanks

**Reply by ralfHielscher** [reply]
What about filling these values with zeros?

Ralf.
